tests/subsys/logging/log_blocking/testcase.yaml

```diff
@@ -0,0 +1,28 @@
+common:
+  filter: CONFIG_QEMU_TARGET and not CONFIG_SMP
+  tags:
+    - log_api
+    - logging
+  integration_platforms:
+    - qemu_x86
+tests:
+  logging.blocking.rate.input_limited:
+    extra_configs:
+      - CONFIG_TEST_INPUT_LOG_RATE=500
+      - CONFIG_TEST_OUTPUT_LOG_RATE=1000
+  logging.blocking.rate.matched:
+    extra_configs:
+      - CONFIG_TEST_INPUT_LOG_RATE=1000
+      - CONFIG_TEST_OUTPUT_LOG_RATE=1000
+  logging.blocking.rate.output_limited:
+    extra_configs:
+      - CONFIG_TEST_INPUT_LOG_RATE=1000
+      - CONFIG_TEST_OUTPUT_LOG_RATE=500
+  logging.blocking.rate.stalled:
+    # Note: this test is expected to "fail".. i.e. the app should
+    # hang. I'm just not entirely sure what the best way is to test
+    # that. Separate thread? ZStress?
```

Comment:
@cfriedt  we currently have `ZTEST_EXPECT_FAIL(<suite_name>, <test_name>)` to support expecting a test case to fail.

Reply:
Fixed!